tests/openstack/requests/image/image_tests.rb

```diff
@@ -1,5 +1,6 @@
 Shindo.tests('Fog::Image[:openstack] | image requests', ['openstack']) do
-  openstack = Fog::Identity[:openstack]
+  openstack = Fog::Identity::OpenStack::V2.new
```

Comment:
I think this will not work when you will run the tests against actutal env, with MOCK=false

Fog::Identity[:openstack] gets actualy a working identity, with credentials from ~/.fog